app/hotels/src/singleHotel/roomList/BeddingInfo.js

```diff
@@ -73,11 +73,13 @@ export class BeddingInfo extends React.Component<Props> {
 
 export default createFragmentContainer(BeddingInfo, {
   room: graphql`
-    fragment BeddingInfo_room on HotelRoomInterface {
-      maxPersons
-      bedding {
-        type
-        amount
+    fragment BeddingInfo_room on HotelRoomAvailabilityInterface {
+      maxOccupancy
+      room {
```

Comment:
It's unfortunate `room` is an attribute of `room` object. It makes reasoning about what's happening slightly confusing. But then for `maxPersons` variable a few lines before, it makes sense. Could you rename the fragment variables to be something else? Though I'm not sure...

Reply:
The parent is `availableRoom` which in the end is a room. There has been some unfortunate naming from the start of this project. Should we follow logic from the provider, there should have been Room and Block. But it makes no sense to start doing this on the client now. 

Maybe our graphql is ready for some refactoring. However this is a major job. Not something to be done anytime soon I guess.

Reply:
I agree, will approve it as it is.